lib/order.rb

```diff
@@ -0,0 +1,66 @@
+require 'twilio-ruby'
+require_relative 'menu'
+
+class Order
+  attr_reader :basket, :menu
+
+  def initialize(menu = Menu.new)
+    @basket = []
+    @menu = menu
+  end
+
+  def add(quantity = 1, item)
+    selection = search(item)
+    add_to_basket(quantity, selection)
+    puts quantity.to_s + ' ' + item + ' added to basket!'
+  end
+
+  def review_order
```

Comment:
In a way, this method has one responsibility - showing a view of the current order.  However, it also has multiple responsibilities because it handles the details of some of the parts of its main responsibility (checking the basket isn't empty and printing the order total).  Could you delegate the details of those responsibilities?